src/main/osd/osd.c

```diff
@@ -1165,11 +1169,12 @@ bool osdUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTimeUs)
 // Called when there is OSD update work to be done
 void osdUpdate(timeUs_t currentTimeUs)
 {
-    static timeUs_t osdStateDurationUs[OSD_STATE_COUNT] = { 0 };
-    static timeUs_t osdElementDurationUs[OSD_ITEM_COUNT] = { 0 };
-    static timeUs_t osdElementGroupMembership[OSD_ITEM_COUNT];
-    static timeUs_t osdElementGroupTargetUs[OSD_GROUP_COUNT] = { 0 };
-    static timeUs_t osdElementGroupDurationUs[OSD_GROUP_COUNT] = { 0 };
+    // Times are known to be short so use uint16_t rather than timeUs_t
```

Comment:
I deleted this comment in my commit because not all the times are short, and there is now one uint32_t which contradicts the comment.

Reply:
Done